postgis/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:latest
```

Comment:
Is this really what we want? This will mean that it won't be possible to know which TimescaleDB version this currently points to.

Note that we always push a "latest" tag, even when we have specific version tag.